views/admin/general/adminLoginContent.cfm

```diff
@@ -0,0 +1,11 @@
+<cfscript>
+   args.bgVideo = args.bgVideo ?: "";
+</cfscript>
+
+<cfif args.bgVideo.len()>
+   <div class="vimeo-wrapper">
```

Comment:
vimeo-wrapper is a bit specific, if it could also be a YouTube video. Maybe something more generic like login-video-wrapper